src/coord/src/coord.rs

```diff
@@ -409,9 +408,10 @@ impl Coordinator {
                                             // before being fully reloaded.
                                             let updates = std::mem::replace(&mut updates, vec![]);
                                             if !updates.is_empty() {
+                                                let mut updates_map = HashMap::new();
```

Comment:
should I rip all of this code out? @danhhz wdyt -- this seems like unnecessary clutter at this point